libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -739,6 +739,7 @@ def similarity_search_with_score_by_vector(
         param: Optional[dict] = None,
         expr: Optional[str] = None,
         timeout: Optional[float] = None,
+        include_id: Optional[bool] = False,
         **kwargs: Any,
```

Comment:
**Static Code Issue**: Dynamically typed expressions (typing.Any) are disallowed in '**kwargs'. <br> **Fix**: Specify explicit types for '**kwargs'. <br> **Code Suggestion**: 
 ```
 -    **kwargs: Any,
 +    **kwargs: dict[str, Any],
 ```